zanata-war/src/main/java/org/zanata/servlet/ZanataDatabaseMetadataServletListener.java

```diff
@@ -23,17 +23,20 @@
 import java.sql.Connection;
 import java.sql.DatabaseMetaData;
 import javax.annotation.Resource;
-import javax.enterprise.inject.Produces;
 import javax.servlet.ServletContextEvent;
 import javax.servlet.ServletContextListener;
 import javax.sql.DataSource;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.zanata.util.ZanataDatabaseDriverMetadata;
-import org.zanata.util.ZanataDatabaseMetaData;
+import org.zanata.util.DatabaseDriverVersionInfo;
+import org.zanata.util.DatabaseVersionInfo;
 
 /**
+ * Log database and database driver information at context start time.
+ * This listener should appear before liquibase listener so that if liquibase
+ * run into any error we can use this information for troubleshooting.
+ *
  * @author Patrick Huang <a href="mailto:pahuang@redhat.com">pahuang@redhat.com</a>
  */
 public class ZanataDatabaseMetadataServletListener implements
```

Comment:
This class has less to do with servlets than it does with databases. I would suggest the package org.zanata.database.